extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js

```diff
@@ -1,8 +1,9 @@
 /*global define*/
 define('ext.wikia.adEngine.lookup.prebid.adapters.appnexus',[
+	'ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements',
 	'wikia.geo',
 	'wikia.instantGlobals'
-], function (geo, instantGlobals) {
+], function (appnexusPlacements, geo, instantGlobals) {
 	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)